include/mbedtls/sha512.h

```diff
@@ -49,98 +48,106 @@ extern "C" {
 #endif
 
 /**
- * \brief          SHA-512 context structure
+ * \brief          The SHA-512 context structure.
```

Comment:
Suggestion: We might want to add here something along the lines of `The structure is used both for SHA-512 and for SHA-384 checksum calculations. The choice between these two is made in the call to mbedtls_md_starts_ret().`

Reply:
do you not mean mean mbedtls_sha512_starts_ret and in SHA-256 mbedtls_sha256_starts_ret instead of mbedtls_md_starts_ret?

Reply:
Yes, sorry, early-morning-copy-paste mistake!

Reply:
Fixed